tests/run-make/arm-target-cpu-fp/rmake.rs

```diff
@@ -0,0 +1,37 @@
+// This tests that target-cpu correctly enables additional floating-point features.
+
+use run_make_support::{llvm_filecheck, llvm_objdump, rustc, static_lib_name};
+
+struct TestCase {
+    target: &'static str,
+    cpu: &'static str,
+}
+
+static CASES: &[TestCase] = &[TestCase { target: "armv8r-none-eabihf", cpu: "cortex-r52" }];
+
+fn main() {
+    for case in CASES {
+        let lib = static_lib_name(case.cpu);
+
+        rustc()
+            .edition("2021")
+            .arg(format!("--target={}", case.target))
+            .arg(format!("-Ctarget-cpu={}", case.cpu))
+            .arg("-Copt-level=3")
+            .crate_type("rlib")
+            .input("lib.rs")
+            .output(&lib)
+            .run();
```

Comment:
@chrisnc Each host that runs the run-make test suite, which includes all tier 1 hosts at least, should execute every rmake test.

Here, you explicitly built the code for each target we are checking cross-compilation against.

So yes, it should be run for each target.

Is there something that seems incomplete about this explanation?

Reply:
Oh, I see, you're talking about the lack of precompiled stds.

Reply:
hrmm, you raise a good point, this will work if we had a tier 2 target we could test this for, because then std will be compiled, but I can't see one...

Reply:
Yeah, I think the expectation is clear, I'm just not sure where to specify in CI that this test needs to be built with `--target=`. It didn't work without this on thumbv8m.main either, which is tier2. I needed to provide `./x test --target=...` in each case.

Reply:
right. CI *will* build the stdlibs for these targets, but locally you won't.

can you set up a variant for thumbv8.main then?

Reply:
Aha. Will do. Is there a way to invoke the test such that this happens automatically?

Reply:
Added. If I include only one case and invoke it with the correct `--target=...`, each of them works. Still not sure how to have the entire test run with all of the cases in one go though.

Reply:
Oh, I missed your earlier comments about tier 2. I will remove the armv8r one. There are tests I can add for thumbv7em and armv7r though...

Reply:
Can you leave a FIXME(#130375) for this? In the meantime, I think it's fine if it's somewhat limited for now.

Reply:
Added a FIXME to the top of rmake.rs, as well as expanding on what the test is for.